src/platform/packages/shared/kbn-unified-metrics-grid/src/components/metrics_grid.tsx

```diff
@@ -8,31 +8,29 @@
  */
 
 import React from 'react';
-import { EuiFlexGrid, EuiFlexItem, EuiLoadingChart, EuiFlexGroup, EuiText } from '@elastic/eui';
+import type { EuiFlexGridProps, IconType } from '@elastic/eui';
```

Comment:
Do you think we should move the `metrics_grid_section` test to test this component instead?

Reply:
I've created some tests. Will push soon

Reply:
[409563d](https://github.com/elastic/kibana/pull/232406/commits/409563d14da78a4b4cdb59fc8b46b4b61b2856cd)